content/millennium/overview.md

```diff
@@ -32,6 +32,6 @@ implementation. For this reason, we highly encourage development against R4 when
 
 ## Supported Resources Between Versions
 
-To determine what resources and actions are supported by our DSTU2 and R4 implementations you can browse our 
-documentation for both DSTU2 and R4 and compare between each version. You can also compare the Conformance (DSTU2) and 
-CapabilityStatement (R4) for the list of supported resources and actions.
\ No newline at end of file
+The table below gives a live view of what resources and actions are currently supported by our DSTU2 and R4 implementations.
```

Comment:
We need to change this to: 
> The table below gives a live view of what resources and actions are currently supported by our DSTU2 and R4 implementations for system and provider access.

Reply:
Updated here: 2f4539944001fa7ffc7a84b8ee0671f2297122be